parser-typechecker/tests/Suite.hs

```diff
@@ -66,7 +65,6 @@ test rt = tests
   , Typechecker.test
   , UriParser.test
   , Context.test
-  , Git.test
```

Comment:
The test relies on specific GUIDs being generated for some types it creates. Should open a separate issue to refactor the test to not do that.

Reply:
Gotcha. Yeah, please create a ticket to track.

Reply:
Tests still need to be able rely on specific GUIDs being generated; refactoring this sort of test to not do that isn't possible, I don't think.

Currently, the transcript runner generates deterministic GUIDs according to the number of `Command`s that have been executed. This PR changed the sequence of `Command`s, which affected the GUID sequence, which changed the "deterministic" sequence, breaking the test's assumption.

We need to open a separate issue to refactor the deterministic GUID generation to depend on the number of typechecks (or something), rather than the number of commands, and then recreate this test to rely on the new, robust, deterministic GUID sequence.